src/tab/Tabs.ts

```diff
@@ -0,0 +1,259 @@
+import { html, css, TemplateResult, nothing } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { OmniElement } from '../core/OmniElement.js';
+import { TabHeader } from './TabHeader.js';
+
+/**
+ * Component that displays content in tabs.
+ *
+ * @import
+ * ```js
+ * import '@capitec/omni-components/tab';
+ * ```
+ *
+ * @example
+ * ```html
+ * <omni-tabs>
+ *  <omni-tab header='Tab 1'>
+ *   <omni-label>Tab 1</omni-label>
+ *  </omni-tab>
+ *  <omni-tab header="Tab 2">
+ *   <omni-label>Tab 2</omni-label>
+ *  </omni-tab>
+ *  <omni-tab header="Tab 3">
+ *   <omni-label>Tab 3</omni-label>
+ *  </omni-tab>
+ * </omni-tabs>
+ *```
+ *
+ * @element omni-tabs
+ *
+ * Registry of all properties defined by the component.
+ *
+ * @fires {CustomEvent<{ previous: HTMLElement, selected: HTMLElement}>} tab-select - Dispatched when an omni-tab is selected.
+ *
+ * @slot - All omni-tab components that are managed by this component.
+ * @slot header - Optional omni-tab-header components associated with each omni-tab component.
+ *
+ * @cssprop --omni-tabs-tab-bar-width - Tabs tab bar width.
+ * @cssprop --omni-tabs-tab-bar-height - Tabs tab bar height.
+ * @cssprop --omni-tabs-tab-bar-border-bottom - Tabs tab bar bottom border.
+ * @cssprop --omni-tabs-tab-bar-background-color - Tabs tab bar background color.
+ *
+ * @cssprop --omni-tab-header-font-color - Tab header component font color.
```

Comment:
Are all header related props correct to list here?

Reply:
listed here to make these css variables visible on the doc site as there is not a stories file for the tab header

Reply:
That's not great, we should extend the docs site to cope with the implementation, rather hack the component to work on the docs site. I did try to bring this up initially when we still had the original components.

Reply:
Added a change to the StoryUtils.ts file to render the css variables of the tab header and tab on the tabs docs.